certified-connectors/jbhunt/README.md

```diff
@@ -0,0 +1,26 @@
+The JBHunt connector with Dynamics 365 Intelligent Order Management provides access to committed, market-relevant rating, tendering, and tracking, connected to nearly 1M trucks through carriers on 360 Marketplace by J.B. Hunt. 
+
```

Comment:
@vechall Please add publisher section here, for more details refer the templates